src/experiments/civic/index.js

```diff
@@ -1,6 +1,6 @@
 export default {
   name: 'civic-register-page-error',
   experimentID: '-ZAD0MyxTySnOmA902g6Jg',
-  isEligible: ({ route }) => !process.server && route.name === 'civic-register',
+  isEligible: () => !process.server,
```

Comment:
if only want to test ppl directly landed on `/civic`, can keep the route check with name `civic`

Reply:
reverted

Reply:
`civic` not `civic-register`

Reply:
this `isEligible` hook only trigger first time ppl arrive the site, so its kind of strange

Reply:
updated